### DISCUSSION TITLE
please ban your admins

### DISCUSSION THREAD

**Discussion by masood151117** [PurpurMC/Purpur#1179] [__Dismissed__] [answered]
beccuse im an iranian teenager and i can not use premium minecraft version . and i ran my server with offline mode . i have been banned by your admins . if you can please tell me how can i buy a minecraft premium without banking sunctions . and if you can not help me so please unban me and ignore my cracked server . beccuse i can not buy minecraft account and i wanna to play with my friends with your software

**Comment by masood151117** [comment]
Masood#6741

**Comment by masood151117** [comment, marked as answer]
and your admins broke my human rights

**Comment by masood151117** [comment]
:x unban me

**Comment by ryderbelserion** [comment]
This definitely is not the way to get unbanned.

**Comment by masood151117** [comment]
:x please

**Comment by masood151117** [comment]
pleaseee

**Comment by masood151117** [comment]
pleaseee

**Comment by YouHaveTrouble** [comment]
I'm locking this and noone dare to delete this. It's gonna get quoted by our grandchildren someday, I guarantee it.

**Comment by BillyGalbreath** [comment]
> i ran my server with offline mode . i have been banned by your admins . beccuse i can not buy minecraft account and i wanna to play with my friends with your software

You are free to do whatever you please. If you don't want (or cant) purchase minecraft, then by all means run your server in offline mode. No one is stopping you from doing so. You just will not receive any support from us (and many other support communities) while doing so.

> and your admins broke my human rights

It is not a right to be a part of our Discord community. __It is a privilege__. We reserve the right to remove anyone at any time for any reason. We have opted to silence/remove offline mode users because we do not support offline mode servers.

To reiterate, you can use offline mode all you want. You just wont be a part of our support community while doing so.
